docs/en/enterprise-edition/content-collections/get-started/access-prisma-cloud.adoc

```diff
@@ -69,12 +75,14 @@ If you see the serial number for your instance and want to change it to a descri
 [#id3c964e17-24c6-4e7c-9a47-adae096cc88d]
 === Log in Using the Prisma Cloud URL
 
-If you have accepted the EULA and have a Palo Alto Networks CSP account, you can access the Prisma Cloud app directly.
+Follow the steps below to access the Prisma Cloud app directly.
 
 [.procedure]
 . Access the Prisma Cloud App.
 +
-Enter your username and password. The URL is *<Prisma Cloud URL>*/auth/signin*
+Navigate to https://appX.prismacloud.io/login (where X is the instance where your tenant is provisioned, for example app3). Use the xref:#domain-list[domain-list] list above for domain specific URLs.
```

Comment:
@jenjoe22 can we change the formatting here to reflect that the URLs don't all follow the 'appx' convention?

Generally we follow a {BASE_URL}/login convention where the BASE_URL in question are those laid out in the linked doc.

Reply:
Does this work @ftbrecordspan?